media/css/m24/components/navigation-refresh.scss

```diff
@@ -263,9 +273,26 @@
                 height: auto;
                 background-color: #fff;
                 overflow: hidden auto;
-                transition: 0.45s;
-                margin-top: 50px;
-                transform: translateX(100%);
+                margin-top: 98px;
+                animation: slide-in 0.45s ease;
+            }
+
+            @keyframes slide-in {
+                from {
+                    transform: translateX(100%);
+                }
+
+                to {
+                    transform: translateX(0);
+                }
+            }
```

Comment:
**suggestion:** there's no need for these animation keyframes to be nested.  We should be able to use one top-level keyframe in this file for both `.c-navigation-items` and `.c-menu-panel` to share? 

Suggest also giving it a component-specific name, such as `nav-slide-in`?